EIPS/eip-5593.md

```diff
@@ -6,7 +6,7 @@ author: Yan Zhu (@diracdeltas), Brian R. Bondy (@bbondy), Andrea  Brancaleoni (@
 discussions-to: https://ethereum-magicians.org/t/rfc-limiting-provider-object-injection-to-secure-contexts/10670
 status: Draft
 type: Standards Track
-category: Interface
+category: ERC
```

Comment:
ERC doesn't seem right here. The group working on ERCs is distinctly different from the group working on wallets and wallet providers and I think we want to make that distinction clear still.